web/Dockerfile

```diff
@@ -17,6 +17,8 @@ WORKDIR /go/src/github.com/runconduit/conduit
 COPY web web
 COPY controller controller
 COPY pkg pkg
+ARG CONDUIT_VERSION
+RUN sed -i "s/unknown/$CONDUIT_VERSION/" pkg/version/version.go
```

Comment:
What do you think about using Go linker's `-X` option:
https://golang.org/cmd/link/

...and then we'd do something like:
```
RUN CGO_ENABLED=0 GOOS=linux go build -ldflags "-X version.Version=$CONDUIT_VERSION" -a -installsuffix cgo -o web/web ./web
```

This could have an added benefit keeping the Go builds more consistent between Docker and native.